app/models/service_template_transformation_plan_task.rb

```diff
@@ -34,6 +34,67 @@ def task_active
     vm_resource.update_attributes(:status => ServiceResource::STATUS_ACTIVE)
   end
 
+  def conversion_host
+    Host.find_by(:id => options["transformation_host_id"])
+  end
+
+  def v2v_log
+    host = conversion_host
+    if host.nil?
+      msg = "Conversion host was not found: ID [#{options["transformation_host_id"]}]. Download of v2v log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    userid, password = host.auth_user_pwd(:default)
+    if userid.blank? || password.blank?
+      msg = "Host credential is required: userid [#{userid}], password [#{password}]. Download of v2v log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    logfile = options.fetch_path("virtv2v_wrapper", "v2v_log")
+    if logfile.blank?
+      msg = "The location of v2v log was not set. Download of v2v log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    begin
+      require 'net/scp'
+      Net::SCP::download!(host.ipaddress, userid, logfile, nil, :ssh => {:password => password})
+    rescue => scp_err
+      _log.error("Download of v2v log for #{description} with ID [#{id}] failed with error: #{scp_err}")
+      raise MiqException::Error, scp_err.to_s, scp_err.backtrace
```

Comment:
Do we need to create an `MiqException::Error` here or can we just re-raise the original error?